apps/testingaccessibility/src/components/team/index.tsx

```diff
@@ -1,28 +1,28 @@
 import React from 'react'
 import SelfRedeemButton from './self-redeem-button'
 import CopyInviteLink from './copy-invite-link'
-import Link from 'next/link'
 
 type InviteTeamProps = {
   purchase: {
     merchantChargeId: string | null
     bulkCoupon: {id: string; maxUses: number; usedCount: number} | null
     product: {id: string; name: string}
   }
-  existingPurchase: {
-    id: string
-    product: {id: string; name: string}
-  }
-  session: any
-  setPersonalPurchase: (props: any) => void
+  existingPurchaseForSelf: boolean
+  userEmail?: string | null
```

Comment:
Tighten the type signature a bit, accept a `userEmail` instead of a whole `session` blob, for instance.